modules/core/include/opencv2/core/cuda.hpp

```diff
@@ -656,6 +656,18 @@ class CV_EXPORTS_W Stream
     //! creates a new asynchronous stream with custom allocator
     CV_WRAP Stream(const Ptr<GpuMat::Allocator>& allocator);
 
+    /** @brief creates a new Stream using the cudaFlags argument to determine the behaviors of the stream
+
+    @note The cudaFlags parameter must be a positive integer; it is passed to the underlying
+    api cudaStreamCreateWithFlags() and supports the same parameter values.
+    @code
+        // creates an OpenCV cuda::Stream that manages an asynchronous, non-blocking,
+        // non-default CUDA stream
+        cv::cuda::Stream cvStream(cudaStreamNonBlocking);
+    @endcode
+     */
+    CV_WRAP Stream(const int cudaFlags);
```

Comment:
I took a look a bit more at bindings generators.
There is `size_t` type support in bindings.

Reply:
I am pushing again to check for any warnings on the CI platforms. Using `size_t` in this way is not in alignment with intention of the type https://en.cppreference.com/w/cpp/types/size_t
It may work on all common platforms/data models today. And it may fail in the future due to using `size_t` in a way the C++ standard did not intend.